### ISSUE TITLE
Does your unmunch work correctly like wordforms command?

### ISSUE THREAD

**Issue by MonsterMMORPG** [eitsl/hunspell#1]
unmunch command of hunspell is not working correctly for other languages such as turkish

the wordforms command works but it doesnt have iterate entire dictionary or export to a file feature

so your unmunch command works or not as wordforms command?

if not working can you add wordforms command so i can use your implementation with iterate and export feature?

ty

**Comment by Gallaecio** [comment]
I don’t think this implementation will work either, and I am currently focusing on other projects and have no plans to implement it.

**Comment by MonsterMMORPG** [comment]
@Gallaecio  how about you implement wordforms command? i think i could easily modify python script 

wordforms : https://github.com/hunspell/hunspell/blob/a6d32ee604e99ffc1919f38664817a2ee1500774/src/tools/wordforms
